### DISCUSSION TITLE
Remove "Do you want to create DFRefactorImportExportFile.json?"

### DISCUSSION THREAD

**Discussion by Caeylinity** [NilsSve/DfRefactor#28] [Ideas]
When navigating folders to get to your DFRefactorImportExportFile.json, it will ask "DFRefactorImportExportFile does not exist. Do you want to create DFRefactorImportExportFile.json?", which is very annoying. I imagine people will save their imports/exports in different places, such as the downloads folder when transferring from person to person or some kind of network drive. 

I also do not see why you would even want to create this file because a blanco version of it is completely useless.

**Comment by NilsSve** [comment]
This is not easily fixed. What is described above, although it certainly can be annoying, is standard Windows behavior for the Microsoft Open dialog.

**Comment by Caeylinity** [comment]
Ah, I wasn't aware that that's Windows defined behaviour, as I've never encountered it before when selecting a file from an explorer window. Maybe it's due to the expectation of finding a file with a specific name, instead of letting the user pick their own file to open? If #26 has been implemented, you wouldn't want to default to DFRefactorImportExportFile.json anyway.

**Comment by NilsSve** [comment]
Yes, that is a fair point. The suggestion for the JSON file is no longer needed and this will (probably) make the Open Dialog box easier to navigate with. The default file setting has now been removed.
